sdk/textanalytics/Azure.AI.TextAnalytics/src/Authoring/TextAuthoringClientOptions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Generally we use the same *ClientOptions class as the inference client. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md and client and clientoptions files under https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src (except in Generated) for how I pulled this off.